### ISSUE TITLE
A request for a small tweak in order to support the JinjaX flavor

### ISSUE THREAD

**Issue by JakobGM** [davidodenwald/prettier-plugin-jinja-template#17]
Hi, @davidodenwald, thanks for the great plugin!

We are using a Jinja2-based component library called [JinjaX](https://jinjax.scaletti.dev/), which almost plays well with `prettier-plugin-jinja-template` besides that `prettier-plugin-jinja-template` ends up quoting variables being passed into the components. That is, a component invocation like this:

```html
<Paginator items={products} />
```

Ends up being reformatted to this:

```html
<Paginator items="{products}" />
```

Which changes the code from passing the context variable `products` into the component as the `items` parameter, to sending the actual literal string `"{products}"` in instead.

Do you think it could be possible to either:

1. Don't quote values that are enclosed by curly brackets.
2. Allow users to disable this specific feature of quoting values.
3. Some other solution?

**Comment by davidodenwald** [comment]
Hey @JakobGM,

I found a similar issue in the prettier project (https://github.com/prettier/prettier/issues/5627) which was solved by adding the parser "lwc" to prettier. This parser removes quotes on attributes if they are surrounded by `{}` and contain no spaces.
Which should solve the problem you are having with this plugin.

I created a branch that adds support for this parser -> https://github.com/davidodenwald/prettier-plugin-jinja-template/pull/22
The option to switch is named "quoteAttributes". 

Feel free to try the feature and provide feedback.
